### PR TITLE
(ToB) PoolUtilInfo htp calculation fix

### PR DESCRIPTION
**Pull request by grandizzy**
# Description of change
## High level
* `PoolUtilsInfo` multiplies twice the max t0 threshold price with pool inflator
  * in `PoolUtilsInfo.poolPricesInfo` and `PoolUtilsInfo.htp` functions do not multiply again the `maxThresholdPrice` value returned from `pool.loansInfo` function (as it's already multiplied)
  * consistent naming of `inflator` across contracts instead `inflatorSnapshot`

### CODE REVIEW

**Review by mattcushman — APPROVED** [review]
lgtm

File: src/PoolInfoUtils.sol [hunk 1]
```diff
@@ -112,14 +112,14 @@ contract PoolInfoUtils {
         (maxBorrower_, , loansCount_) = pool.loansInfo();
 
         (
-            uint256 inflatorSnapshot,
-            uint256 lastInflatorSnapshotUpdate
+            uint256 inflator,
```
Comment (MikeHathaway) [inline src/PoolInfoUtils.sol:115]: Should this also be consistent here? we were using `poolInflator` above, but just `inflator` here

**Review by MikeHathaway — COMMENTED** [review]
Change looks good, agree renaming would be helpful. Just had one question about some of the new names

File: src/PoolInfoUtils.sol (hunk 1 above)
Comment (grandizzy) [reply src/PoolInfoUtils.sol:115]: yep, choose to stick with `inflator` instead `poolInflator` as we have `pendingInflator` / `lastInflatorUpdate` as well, see https://github.com/ajna-finance/contracts/pull/685/commits/b8f2450693c46967f21287165d4824c1281e744d

**Review by MikeHathaway — APPROVED** [review]
LGTM

**Pull request merged**